model/vm.rb

```diff
@@ -207,7 +208,7 @@ def params_json(swap_size_bytes)
       "local_ipv4" => local_vetho_ip.to_s.shellescape || "",
       "dns_ipv4" => nics.first.private_subnet.net4.nth(2).to_s,
       "unix_user" => unix_user,
-      "ssh_public_keys" => [public_key] + project_public_keys,
+      "ssh_public_keys" => public_keys + project_public_keys,
```

Comment:
```suggestion
      "ssh_public_keys" => public_keys,
```

Assuming previous recommended change implemented.